openapi/paths/payout-requests@{id}.yaml

```
@@ -24,3 +24,41 @@ get:
       $ref: ../components/responses/Forbidden.yaml
     404:
       $ref: ../components/responses/NotFound.yaml
+put:
+  x-products:
+    - Core
+  tags:
+    - Transactions
+  summary: Upsert a payout request
+  operationId: PutPayoutRequest
+  x-sdk-operation-name: update
+  description: Create or update (upsert) a payout request with specified identifier string.
```

nitpick: based on style
```suggestion
  description: Creates or updates (upserts) a payout request with specified a ID.
```